Source/vtkPyFR/vtkPyFRPipeline.cxx

```diff
@@ -645,31 +735,35 @@ int vtkPyFRPipeline::CoProcess(vtkCPDataDescription* dataDescription)
     this->InsituLink->InsituUpdate(dataDescription->GetTime(),
                                    dataDescription->GetTimeStep());
 
-    vtkUpdateFilter(this->Contour, dataDescription->GetTime());
-    vtkUpdateFilter(this->Slice, dataDescription->GetTime());
+      vtkUpdateFilter(this->Contour, dataDescription->GetTime());
+      vtkUpdateFilter(this->Slice, dataDescription->GetTime());
+
     if(this->PyData(dataDescription)->PrintMetadata()) {
-        double* bds = this->ContourMapper->GetBounds();
+        double* bds = this->ActiveMapper1->GetBounds();
```

Comment:
is this the only purpose of ActiveMapper? I have not done GetBounds of ActiveMapper2. Is that necessary or is it only for message?